packages/react-native/src/primitives/TextField/TextField.tsx

```diff
@@ -38,19 +39,28 @@ export default function TextField({
 
   return (
     <View
-      accessible={accessible}
-      accessibilityLabel={accessibilityLabel ?? label}
-      accessibilityRole={accessibilityRole}
-      accessibilityState={{ disabled, ...accessibilityState }}
```

Comment:
There was a misconception about what the outcome of this would be. Adding accessibility props on a parent View basically means a single label will be announced for all children. That meant the text field accessory would not get announced by voiceover.